src/applications/vaos/services/appointment/index.js

```diff
@@ -119,47 +119,66 @@ function hasPartialResults(response) {
 export async function getBookedAppointments({
   startDate,
   endDate,
-  useV2 = false,
+  useV2VA = false,
+  useV2CC = false,
 }) {
   try {
-    if (useV2) {
-      const appointments = await getAppointments(startDate, endDate, [
-        'booked',
-        'cancelled',
-      ]);
-
-      const appointmentsWithoutRequests = appointments.filter(
-        appt => !appt.requestedPeriods,
+    let appointments = [];
```

Comment:
This logic is a little weird with the both feature flags so I reorganized it a bit.